### ISSUE TITLE
[WCAG 2.2 Draft Feedback] Success Criterion 2.4.13 Fixed Reference Points

### ISSUE THREAD

**Issue by dshoukry** [w3c/wcag#1449]
“Success Criterion [2.4.13 Fixed Reference Points](https://www.w3.org/TR/WCAG22/#fixed-reference-points) (Level A): When a [web page](https://www.w3.org/WAI/WCAG22/Understanding/fixed-reference-points.html#dfn-web-page) or [set of web pages](https://www.w3.org/WAI/WCAG22/Understanding/fixed-reference-points.html#dfn-set-of-web-pages) is an [electronic publication](https://www.w3.org/WAI/WCAG22/Understanding/fixed-reference-points.html#) with [pagebreak locators](https://www.w3.org/WAI/WCAG22/Understanding/fixed-reference-points.html#), a mechanism is available to navigate to each locator and each locator maintains its place in the flow of content, even when the formatting or platform change.”

We kindly request significant changes to this SC, and if those can't be made we would recommend that the group remove this SC or point readers to [EPUB Requirement 4.4.1 Page Navigation](http://idpf.org/epub/a11y/#sec-page-navhttps://www.w3.org/Submission/epub-a11y/#sec-page-nav). The intent of this SC is excellent, but the wording around it, especially in the opening example and even in the description, feels more like it is specific to something like an EPUB Requirement 4.4.1 Page Navigation. There are many scenarios on the web where there are pagers and other ways of breaking up content, and when referring to sections of the content, a concept of physical location might be used instead of digital context. (Other non-page related examples: to the right, above, below, etc.)

This also seems like an especially important area to have research, a diverse set of use cases, and a specific set of content structures to keep the SC beneficial, appropriately scoped, and attainable. 

Please find detailed specifics covered in our [2.4.13 Fixed Reference Points (Level A) Google Doc](https://docs.google.com/document/d/1oF0ZN5tRZAsCLfB2GGvRMRTISl0VrCP4485a2__fffg/edit#heading=h.23ybyllde921).

**Comment by dshoukry** [comment]
As requested, I've just replaced the Google Doc link with a version with expanded access (didn't make any content changes). Sorry for the access issues with the previous link.

**Comment by alastc** [comment]
Hi @dshoukry,

Regarding your main point:

> The intent of this SC is excellent, but the wording around it, especially in the opening example and even in the description, feels more like it is specific to something like an EPUB. There are plenty of scenarios on the web where there are pagers and other ways of breaking up content, and when referring to sections of the content, a concept of physical location might be used instead of digital context. (Other non-page related examples: to the right, above, below, etc.)
> 
> This seems like an especially important area to have research, a diverse set of use cases, and a specific set of content structures to keep the SC beneficial, appropriately scoped, and attainable.
> 
> Can the wording and examples be framed more around the way that we think of web browsing and digital content, and less of how we perceive a book?
> 
> Otherwise, can this SC, which seems scoped to electronic publications, be removed or updated to point readers to EPUB Requirement 4.4.1 Page Navigation?http://idpf.org/epub/a11y/#sec-page-navhttps://www.w3.org/Submission/epub-a11y/#sec-page-nav

This requirement was brought to the AG group by the ePub group, they were asking for this to be included (along with another one which didn't seem to fit well enough).

Whilst the requirement for this criterion is well established in an ePub context, we are conscious that we don't have sufficient knowledge/research about a more general requirement. Therefore it is hard to create a more general requirement.

As ePub documents (when presented in a web context) are in scope of the guidelines it is appropriate to include something specific to ePub. We can happily point to the ePub requirement in the understanding document, but without the SC there is nowhere to point readers to an ePub requirement.

**Comment by alastc** [comment]
The [associated PR](https://github.com/w3c/wcag/pull/1582/files) includes:
- Renaming the technique to "Providing a list of pages".
- Editorial changes to the intent section.
- Replacing "paper" with "print" in the example.
- Adding the ePub spec in the resources.

**Comment by mraccess77** [comment]
I have concerns with the new proposed title of "providing a list of pages" as the title implies it could be met by just doing that without actually putting markers into the content letting the user know the corresponding print page from where they currently are reading.  The need is 2 address both issues of getting to the print page and knowing the corresponding page while reading.

**Comment by alastc** [comment]
Hi Jon,

The comment being addressed was just a grammar one, where "provide a page list" should be "pages list". That sounds odd to me, so I updated it to "provide a list of pages".

I don't think that changes the meaning, but if you have a better idea please let me know.

**Comment by dshoukry** [comment]
Thanks for the updates and for accepting some of the suggestions. I'm gathering internal feedback about the main point and different publication versions responses and will provide an update soon.

**Comment by alastc** [comment]
A new draft response for the group to consider:
<hr>

This requirement was brought to the AG group by the ePub group, they were asking for this to be included (along with another one which didn't seem to fit well enough).

Whilst the requirement for this criterion is well established in an ePub context, we are conscious that we don't have sufficient knowledge/research about a more general requirement. Therefore it is hard to create a more general requirement.

As ePub documents (when presented in a web context) are in scope of the guidelines it is appropriate to include something specific to ePub. We can happily point to the ePub requirement in the understanding document, but without the SC there is nowhere to point readers to an ePub requirement.

Please also note that there is a significantly [updated version](https://w3c.github.io/wcag/understanding/fixed-reference-points.html) now which focuses the requirement on providing a navigation mechanism _**if**_ page break locators are provided.

The associated PR includes suggested changes:

- Renaming the technique to "Providing a list of pages".
- Editorial changes to the intent section.
- Replacing "paper" with "print" in the example.

**Comment by alastc** [comment]
The response above was accepted by the group today. There will be another round of reviews due to significant changes.
